doc/source/whatsnew/v0.24.0.txt

```diff
@@ -1192,7 +1192,7 @@ Numeric
 Strings
 ^^^^^^^
 
--
+- BUG Index.str.partition not nan-safe (:issue:`23558`)
```

Comment:
can you use ``:func:`Index.str.partition` ``, and format like the other issues

Reply:
I tried to follow the style of the other messages in the next commit. I hope it's correct now.